### DISCUSSION TITLE
[Alpha] NotificationBadge

### DISCUSSION THREAD

**Discussion by sjur-gr** [navikt/Designsystemet#2581] [Komponenter]
## Velkommen til vurdering av NotificationBadge 🙌
`NotificationBadge` brukes til å markere at det har kommet ny informasjon. For eksempel nye meldinger eller en oppdatert sak.

Forslaget her er å tilby en måte å markere at det har kommet ny informasjon. Markeringen skal være tydelig for brukerne. Samt ha støtte for å vise antall.
<img width="656" alt="Skjermbilde 2023-12-22 kl  08 12 28" src="https://github.com/navikt/aksel/assets/82579704/d046d157-c4de-44f3-ab00-623aba14be54">

Vi har lagt et eksempel på komponenten [NotificationBadge i Alpha UI kit](https://www.figma.com/file/Jw3DCrpJgK0uiG7v92Wdnk/Alpha-UI-kit?type=design&node-id=571-27889&mode=design) som dere kan teste ut.

For å finne ut om vi skal inkludere `NotificationBadge` som en komponent i Aksel trenger vi deres hjelp. **Vi er på jakt etter bevis på behov i løsninger som er i prod/demo**. Vi ønsker skjermbilder (uten sensitive opplysninger) og en forklaring av konteksten.

Vi holder ikke aktivt øye med denne diskusjonen, så gi oss et hint om du legger til noe her. Når vi har **fått bevis på minst 3 tilfeller av bruk** vil vi gå videre med komponenten. Del dette gjerne med team du tror `NotificationBadge` vil passe for.

**Comment by martineeklund** [comment]
Vi bruker notification badge i saksbehandlingssystem for hjelpemidler (Hotsak), for å vise at det ligger utkast til notater som må ferdigstilles (til høyre i skjermbildet). 
![image](https://github.com/user-attachments/assets/3eefe845-1903-4d9f-94f3-9cfc4c0859b7)
